sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -16,26 +15,25 @@ goog.scope(function() {
 
     glsFboCompletenessTests.initGlDependents = function(gl) {
         if (!(gl = gl || window.gl)) throw new Error('Invalid gl object');
-        
-        
+
         // The following extensions are applicable both to ES2 and ES3.
         /**
-         * GL_OES_depth_texture
+         * gl.OES_depth_texture
```

Comment:
These comment renames look wrong. WebGL drops the GL_ prefix from extension names but doesn't define them as properties on the WebGLRenderingContext.